enhancements/installer/aws-lb-subnet-selection.md

```diff
@@ -0,0 +1,1147 @@
+| ClusterNode            | Host the cluster nodes, including both control plane and compute nodes in standard availability zones (excluding local zones).                                                                                                                                                                                                   | Private [^1]                                                |
+| EdgeNode               | Host nodes running in AWS edge zones, Local Zones or Wavelength Zones. These subnets are exclusively used for hosting edge nodes and cannot attach to Control Plane or IngressController load balancers.                                                                                                                         | Private [^1]                                                |
+| Bootstrap          | Host the bootstrap resources such as a temporary machine booted to act as a temporary control plane whose sole purpose is launching the rest of the cluster ([reference](https://github.com/openshift/installer/blob/main/docs/user/overview.md#cluster-installation-process)).                                                  | Public or Private                                           |
+| IngressControllerLB    | Host AWS load balancer created specifically for the `default` IngressController. This role does not include load balancers created from generic LoadBalancer-type Services. Defaulting for user-created IngressControllers is not included for this role; however, this role may be extended in the future include such support. | Public for External Clusters, Private for Internal Clusters |
+| ControlPlaneExternalLB | Host the control plane's **external** load balancer that serves the Kubernetes API server. Only created for externally published clusters.                                                                                                                                                                                       | Public                                                      |
+| ControlPlaneInternalLB | Host the control plane **internal** load balancers that serves the Kubernetes API server. Always created for all clusters.                                                                                                                                                                                                       | Private                                                     |
+[^1]: Can be public for OPENSHIFT_INSTALL_AWS_PUBLIC_ONLY
+
+_Note: These roles are defined only to establish a common language within the context of the install-config
+and may not be applicable to other environments, situations, or APIs._
+
+### Clarifying Subnet Roles Terminology
+
+For simplicity, "IngressControllerLB subnets," "ControlPlaneExternalLB subnets," "ControlPlaneInternalLB subnets",
```

Comment:
```suggestion
This proposal is designed to take care of subnets serving multiple roles. 
```
We don't need to list the roles again here, reducing the burden of updating in multiple places if the list changes or their names change. I hope the suggestion provided doesn't take away from the original intent.

Reply:
I think this is a matter of personal opinion, I tend to find being more verbose, as the text is currently, reduces potential for ambiguity. Either is fine though